tests/test_api.py

```diff
@@ -1840,18 +1851,21 @@ def __init__(self, platform, release, platforms, arrangement_version,
             'scratch': None,
             'platforms': platforms,
             'release': release,
-            'git_uri': None,
-            'git_ref': None,
-            'git_branch': None,
+            'git_uri': TEST_GIT_URI,
+            'git_ref': TEST_GIT_REF,
+            'git_branch': TEST_GIT_BRANCH,
             'user': None,
             'tag': None,
             'target': None,
             'architecture': None,
             'yum_repourls': None,
         }
-        if worker:
+        if arrangement_version:
             expected_kwargs.update({
                 'arrangement_version': arrangement_version,
+            })
+        if koji_upload_dir:
```

Comment:
https://github.com/projectatomic/osbs-client/blob/master/osbs/cli/main.py#L384

koji_upload_dir is passed ONLY when is defined